pom.xml

```diff
@@ -210,7 +210,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials</artifactId>
-      <version>1.16.1</version>
+      <version>2.1.12</version>
```

Comment:
* `SecretBytes` is since credentials-2.1.5.
* Stapler bindings of `SecretBytes` (that is, using with `DataBoundConstructor` and `DataBoundSetter`) is since credentials-2.1.12